src/openapi.yaml

```diff
@@ -5309,6 +5495,19 @@ paths:
             X-Is-Form-Designer:
               $ref: '#/components/headers/X-Is-Form-Designer'
             Content-Language:
+    ApiTypeEnum:
```

Comment:
Not sure where these came from :thinking:

Reply:
Seems to have been resolved after rebasing on master and regenerating the schema :man_shrugging: